backend/src/main/java/ddangkong/domain/balance/room/RoomContent.java

```diff
@@ -43,6 +44,36 @@ public class RoomContent extends BaseEntity {
     @Column(nullable = false)
     private boolean isUsed;
 
+    public static RoomContent createNewRoom(Room room, BalanceContent balanceContent, int round) {
```

Comment:
사소한 의견) 
```suggestion
    public static RoomContent createNewRoomContent(Room room, BalanceContent balanceContent, int round) {
```

질문)
[게임 시작 API PR](https://github.com/woowacourse-teams/2024-ddangkong/pull/122/files#diff-d332bcf8aaa7431a1988b2d311c2404ae84f31b9796bc9ff4c88071d0e3da89a)에 있는 `createList()`와 쓰임이 다른가요?

Reply:
애초에 위 정팩메가 테스트에 사용하네요~ 그냥 생성자를 public 하게 사용하도록 바꿀께요~